stack-nightly.yaml

```diff
@@ -21,3 +21,5 @@ extra-deps:
 - http2-client-grpc-0.8.0.0
 - avro-0.4.5.4
 - language-protobuf-1.0
+- github: jaspervdj/stylish-haskell
```

Comment:
These should be removed. Instead, please write a `development.md` file in which you explain that we are using an unreleased branch of `stylish-haskell`.